doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -30,6 +30,12 @@ Changelog
 
 The following sections provide detailed lists of changes by component.
 
+MCUboot
+======
+
+* MCUboot will fail to boot when external flash is used for non-primary application images, if driver for external flash is not enabled.
```

Comment:
I guess we can re-phrase this now that we have the `depends on PM_EXTERNAL_FLASH_HAS_DRIVER` in place.

```suggestion
* Added dependency so that build will fail if external flash is used for secondary slot but no driver for external flash is enabled.
```

Do you agree @de-nordic ?

Reply:
```suggestion
* Added the :kconfig:option:`PM_EXTERNAL_FLASH_HAS_DRIVER` kconfig option.
  When set, it makes MCUboot fail to boot when the external flash memory is used for non-primary application images but the driver for the external flash memory is not enabled.
```